packages/suite/src/actions/suite/metadataLabelingActions.ts

```diff
@@ -583,7 +583,8 @@ export const init =
         }
 
         // 1. set metadata enabled globally
-        if (!getState().metadata.enabled) {
```

Comment:
I wasn't sure if putting this `dispatch(metadataActions.enableMetadata());` to the end of hte proces wouldn't break something, since it is possible that some intermediat processing relies on this being on. So I did a "rollback".